src/main/perl/History.pm

```diff
@@ -0,0 +1,394 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+
+package CAF::History;
+
+use strict;
+use warnings;
+
+use LC::Exception qw (SUCCESS);
+use Readonly;
+
+use parent qw(Exporter);
+our @EXPORT_OK = qw($EVENTS $IDX $ID $TS $REF);
+
+# refaddr was added between 5.8.0 and 5.8.8
```

Comment:
Should this line have a "use 5.8.8" line in it to help catch versions of Perl that ar too old?